content/docs/iac/concepts/resources/dynamic-providers.md

```diff
@@ -773,19 +785,29 @@ class GithubLabelArgs(object):
         self.description = description
 
 class GithubLabelProvider(ResourceProvider):
+    auth: str
+    def configure(self, req: ConfigureRequest):
+        # Set a stack config variable named githubToken (e.g. pulumi config set githubToken <VALUE> --secret)
+        self.auth = req.config.require("githubToken")
+
     def create(self, props):
+        auto_secret = True
+        g = Github(auth.get())
         l = g.get_user(props["owner"]).get_repo(props["repo"]).create_label(
             name=props["name"],
             color=props["color"],
             description=props.get("description", GithubObject.NotSet))
         return CreateResult(l.name, {**props, **l.raw_data})
     def update(self, id, _olds, props):
+        auto_secret = True
+        g = Github(auth.get())
```

Comment:
```suggestion
        g = Github(self.auth.get())
```

Reply:
Oops. Forgot to propagate the self.auth bit.
Changed to `self.auth...`